.github/workflows/test.yml

```diff
@@ -38,27 +48,48 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install
-        run: |
-          pip install --upgrade pip setuptools
-          pip install -e .[all]
-          pip install pytest-xdist
+        run: pip install -e .[all]
 
-      - name: Style test
-        run: flake8 metomi/isodatetime
+      - name: Style
+        run: flake8
 
       - name: Run tests
         env:
           TZ: ${{ matrix.tz }}
-        run: |
-          PYTEST_ARGS=(-n 2 -m 'slow or not slow')
-          if ${{ matrix.coverage }}; then
-            PYTEST_ARGS+=('--cov=metomi/isodatetime')
-          fi
-
-          pytest "${PYTEST_ARGS[@]}"
+          PYTEST_ADDOPTS: ${{ matrix.coverage && format('{0} --cov=metomi/isodatetime', env.PYTEST_ADDOPTS) || env.PYTEST_ADDOPTS }}
+        run: pytest
 
-      - name: Upload coverage report
+      - name: Coverage report
         if: matrix.coverage
         run: |
-          coverage xml --ignore-errors
-          bash <(curl -s https://codecov.io/bash)
+          coverage xml
+          coverage report
```

Comment:
Do we need this bit?

Reply:
`coverage xml` - I think this is needed to convert the data into a compatible format for Codecov? But not sure.

`coverage report` - Not strictly needed but can be useful to see the coverage stats sometimes